resources/assets/js/components/FriendsList.vue

```diff
@@ -129,13 +129,16 @@
                     text: 'This will delete @' + friend.username + ' from your list of friends',
                     type: 'warning',
                     showCancelButton: true
-                }, function () {
+                }, () => {
                     vm.$http.delete('/api/conferences/' + vm.conferenceId + '/' + vm.key + '/' + friend.id)
-                        .then(function () {
-                            vm.list.$remove(friend)
-                        });
+                        .then(() => { vm.list.$remove(friend) });
                 });
             },
+
+            metFriend: function (friend) {
+                this.$http.patch('/api/conferences/' + this.conferenceId + '/' + this.key + '/' + friend.id, { met: ! friend.met })
```

Comment:
Right. But you have a method named "metFriend". Would you be surprised if you called "metFriend" and the end result was that the friend was marked as "not met"?

If I saw a method named "metFriend" I would expect to see something like this:

```javascript
            metFriend: function (friend) {
                this.$http.patch('/api/conferences/' + this.conferenceId + '/' + this.key + '/' + friend.id, { met: true })
                    .then(() => { friend.met = true; });
             }

```